doc/Porting_v1_Capsules_to_v2.md

```diff
@@ -272,70 +273,44 @@ method swaps the passed `ReadOnlyAppSlice` and the one in the `App` region,
 returning the one that was in the app region. It then returns `slice`,
 which is either the passed slice or the swapped out one.
 
-### Example of `subscribe`
+### The new subscription mechanism
 
-A call to `subscribe` has a similar structure to `allow`. Here is
-an example from `console`:
+To use upcalls a capsule now provides the number of upcalls as a templated value
```

Comment:
Explain why, how V2 subscribe is different.

Reply:
Updated.